src/components/RepoList.tsx

```diff
@@ -42,62 +42,64 @@ const RepoList = ({ data }: RepoListProps): JSX.Element => {
 
   return (
     <div className="flex flex-col gap-y-[20px] md:flex-row bg-white border-[1px] p-[16px] gap-x-[20px] font-Inter border-borderGrey overflow-hidden rounded-[16px]">
-      <div>
-        <div className="rounded-[8px] overflow-hidden w-[88px] h-[88px]">
+      <div className="flex gap-y-[20px] gap-x-[20px] font-Inter overflow-hidden w-full">
```

Comment:
Can you instead `gap-y-[20px]`  use `gap-y-5` and instead `gap-x-[20px]` use `gap-x-5'

Reply:
Sure thing, I also adjusted similar classes in the parent div as well.